packages/builders/src/messages/embed/Embed.ts

```diff
@@ -250,36 +278,51 @@ export class EmbedBuilder {
 	 * @param url - The image URL to use
 	 */
 	public setImage(url: string | null): this {
-		// Data assertions
-		imageURLPredicate.parse(url);
-
 		this.data.image = url ? { url } : undefined;
 		return this;
 	}
 
+	/**
+	 * Clears the image of this embed.
+	 */
+	public clearImage(): this {
+		this.data.image = undefined;
+		return this;
+	}
+
 	/**
 	 * Sets the thumbnail of this embed.
 	 *
 	 * @param url - The thumbnail URL to use
 	 */
 	public setThumbnail(url: string | null): this {
-		// Data assertions
-		imageURLPredicate.parse(url);
-
 		this.data.thumbnail = url ? { url } : undefined;
 		return this;
 	}
 
+	/**
+	 * Clears the thumbnail of this embed.
+	 */
+	public clearThumbnail(): this {
```

Comment:
This is redundant, no? setThumbnail supports unsetting

Reply:
That's an accident, it shouldn't